assets/src/dashboard/app/views/templateDetails/content/detailsGallery/index.js

```diff
@@ -19,28 +19,53 @@
 import PropTypes from 'prop-types';
 import { useMemo } from 'react';
 import { sprintf, __ } from '@web-stories-wp/i18n';
+import styled from 'styled-components';
 
 /**
  * Internal dependencies
  */
 import {
-  CardGallery,
-  ColorList,
-  PaginationButton,
-  Pill,
-} from '../../../../../components';
-import { TemplatePropType } from '../../../../../types';
-import {
-  ByLine,
-  Column,
-  ColumnContainer,
-  DetailContainer,
-  LargeDisplayPagination,
-  MetadataContainer,
-  SmallDisplayPagination,
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+  BUTTON_VARIANTS,
+  Chip,
+  Display,
+  Icons,
   Text,
-  Title,
-} from './../../components';
+  THEME_CONSTANTS,
+} from '../../../../../../design-system';
+import { CardGallery, ColorList } from '../../../../../components';
+import { TemplatePropType } from '../../../../../types';
+import { Column, ColumnContainer, DetailContainer } from './../../components';
+
+const ByLineText = styled(Text)`
+  color: ${({ theme }) => theme.colors.fg.tertiary};
+  margin: 8px 0 24px;
+`;
+const DescriptionText = styled(Text)`
+  margin-bottom: 24px;
+`;
+
+const PaginationContainer = styled.div`
+  display: flex;
+  align-items: center;
+  ${({ rightPadding }) =>
+    rightPadding ? 'padding-right: 52px;' : 'padding-left: 52px;'}
+`;
+
+const TemplateTag = styled(Chip)`
+  margin-right: 12px;
+  > span {
+    color: ${({ theme }) => theme.colors.fg.primary} !important;
```

Comment:
These are disabled chips but they aren't supposed to look like it so i'm overriding the span color